docs/sample_config.yaml

```diff
@@ -1502,6 +1502,12 @@ room_prejoin_state:
    #additional_event_types:
    #  - org.example.custom.event.type
 
+# If enabled, puppeted user IP's can also be tracked. By default when
+# puppeting another user, the user who has created the access token
+# for puppeting is tracked. If this is enabled, both requests are tracked.
+# Implicitly enables MAU tracking for puppeted users.
+#track_puppeted_user_ips: false
```

Comment:
* this is missing a `#` line
* I'm struggling to follow the description. What does the first sentence mean? What does "If this is enabled" mean (just uncommenting the example setting, ie setting `track_puppeted_user_ips: false`?)
* convention is that the example should be the *opposite* to the default, which doesn't seem to be the case here.

More details at https://matrix-org.github.io/synapse/develop/code_style.html#configuration-file-format